drivers/led/pca_common.c

```diff
@@ -20,6 +20,10 @@ LOG_MODULE_REGISTER(pca_common);
 
 #include "led_context.h"
 
+#define _DT_PROP_ANY_OR(id, compat, prop) DT_PROP(DT_INST(id, compat), prop) ||
+#define DT_PROP_ANY(compat, prop) \
```

Comment:
The `DT_` namespace is reserved for zephyr/devicetree.h and its sub-headers. Please choose a different name.